Geometry/DTGeometryBuilder/test/runTest.sh

```diff
@@ -5,7 +5,7 @@ function checkDiff {
     FSIZE=$(stat -c%s "$1")
     echo "The output diff is $FSIZE:"
     cat $1;
-    if [ $FSIZE -gt 1000 ]
+    if [ $FSIZE -gt 2000 ]
```

Comment:
@ianna @cvuosalo is this really needed? Please see the solution adopted after the discussion in https://github.com/cms-sw/cmssw/pull/29841 I would say the situation here should be perfectly parallel, no?

Reply:
@fabiocos - good point. I agree, that this test should be revised. I hope, @slomeo could have a look at it at some point and improve the tests :-)

Reply:
@ianna in my experience it was enough to remove the MessageLogger statistics for the output o be compared, so it might be simple (to be cross-checked)

Reply:
@ianna @fabiocos after RPC (PR #30121) and CSC (PR #30122) I'll start with DT. @fabiocos what is the adopted solution?

Reply:
@fabiocos : is this (#29841 )the solution?

Reply:
@slomeo as we discussed yes, it is